src/pages/networks/forms/NetworkAclSelector.tsx

```diff
@@ -0,0 +1,44 @@
+import type { FC } from "react";
+import { MultiSelect } from "@canonical/react-components";
+import { useNetworkAcls } from "context/useNetworkAcls";
+
+interface Props {
+  project: string;
+  setSelectedAcls: (acls: string[]) => void;
+  selectedAcls: string[];
+}
+
+const NetworkAclSelector: FC<Props> = ({
+  project,
+  setSelectedAcls,
+  selectedAcls,
+}) => {
+  const { data: availableAcls = [] } = useNetworkAcls(project);
+
+  const toOptionList = (list: string[]) => {
+    return list.map((item) => {
+      return {
+        label: item,
+        value: item,
+      };
+    });
+  };
+
+  const hasAcls = availableAcls.length > 0;
+
+  return (
+    <MultiSelect
+      items={toOptionList(availableAcls.map((acl) => acl.name))}
+      disabled={!hasAcls}
+      selectedItems={toOptionList(selectedAcls)}
+      variant={hasAcls ? "condensed" : "search"}
+      placeholder={hasAcls ? "Select ACLs" : "No ACLs available"}
+      onItemsUpdate={(items) => {
+        setSelectedAcls(items.map((item) => item.label));
```

Comment:
```suggestion
        setSelectedAcls(items.map((item) => item.value));
```

Although it's the same it's probably more clearer?

Reply:
The typesystem is bamboozled by value, because it can be numeric. But we can cast it here. Agree value is the better option.